migration-challenge/migration-guide.md

```diff
@@ -37,9 +37,9 @@ However, this has a couple of implications that will need you to migrate your co
 
 * v0.15 drops `purs bundle` and relies on an external bundlers
   
-  Yes, you heard right. The Purescript compiler no longer comes with a built-in `bundle` command. Updating `purs bundle` to ESM would have required considerable ammount of work, while at the same time being buggy, doing a poor job on bundle size and taking time from the compiler time to work on more useful things.
+  Yes, you heard right. The Purescript compiler no longer comes with a built-in `bundle` command. `purs bundle` was already broken in a couple of ways, didn't do great job on bundle size and was basically unmaintained. Updating `purs bundle` to ESM would have required a considerable ammount of work, taking time from the compiler team to work on more urgent matters in the compiler.
```

Comment:
```diff
- Yes, you heard right. The Purescript compiler no longer comes with a built-in `bundle` command. `purs bundle` was already broken in a couple of ways, didn't do great job on bundle size and was basically unmaintained. Updating `purs bundle` to ESM would have required a considerable ammount of work, taking time from the compiler team to work on more urgent matters in the compiler.
+ Yes, you heard right. The Purescript compiler no longer comes with a built-in `bundle` command. `purs bundle` was already broken in a couple of ways, didn't do a great job on bundle size, and was basically unmaintained. Updating `purs bundle` to ESM would have required a considerable amount of work, taking time away from the compiler team to work on more urgent matters in the compiler.
```

Reply:
I believe this should present an option on your side to commit this suggestion? It doesn't on mine.

Reply:
I also don't see an commit option, usually it does 🤔

Reply:
Ok changed it accordingly